### DISCUSSION TITLE
command line option to run the interactive-only agent with a custom name instead of the default network/computer name

### DISCUSSION THREAD

**Discussion by abdeldjalil-fellah** [Ylianst/MeshCentral#6043] [General]
how to run the interactive-only agent with a custom name instead of the default network/computer name?

**Comment by si458** [comment]
the only way of doing this would be:
1. download the meshagent as interactive mode and put the file somewhere in a folder
2. go to your meshcentral ui but add `&debug=1` in the URL
3. go to `add agent`, select the `interactive only` then save the `settings file` in the same folder as the exe you downloaded before
4. edit the `.msh` file and add the line `agentName=MYRANDOMNAME`
5. then run the exe and it should read the `.msh` file from its current folder

**Reply by abdeldjalil-fellah** [reply]
what a bout adding a cli option `--agentName` that override the .msh agentName attribute?

**Reply by si458** [reply]
You would need to open a request in the agent repo for something like this as its not that simple as far as im aware https://github.com/Ylianst/MeshAgent
